new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItUsabilityOperatorHelmChart.java

```diff
@@ -0,0 +1,309 @@
+  private static String opNamespace = null;
+  private static String domainNamespace = null;
+  private static HelmParams nginxHelmParams = null;
+  private static int nodeportshttp = 0;
+
+  // domain constants
+  private final String domainUid = "domain1";
+  private final String clusterName = "cluster-1";
+  private final int managedServerPort = 8001;
+  private final int replicaCount = 2;
+  private final String adminServerPodName = domainUid + "-" + ADMIN_SERVER_NAME_BASE;
+  private final String managedServerPrefix = domainUid + "-" + MANAGED_SERVER_NAME_BASE;
+
+  /**
+   * Get namespaces for operator, domain1 and NGINX.
+   * Install and verify NGINX.
+   *
+   * @param namespaces list of namespaces created by the IntegrationTestWatcher by the
+   *                   JUnit engine parameter resolution mechanism
+   */
+  @BeforeAll
+  public static void initAll(@Namespaces(3) List<String> namespaces) {
+
+    // get a unique operator namespace
+    logger.info("Getting a unique namespace for operator");
+    assertNotNull(namespaces.get(0), "Namespace list is null");
+    opNamespace = namespaces.get(0);
+
+    // get a unique domain namespace
+    logger.info("Getting a unique namespace for WebLogic domain");
+    assertNotNull(namespaces.get(1), "Namespace list is null");
+    domainNamespace = namespaces.get(1);
+
+    // get a unique NGINX namespace
+    logger.info("Getting a unique namespace for NGINX");
+    assertNotNull(namespaces.get(2), "Namespace list is null");
+    String nginxNamespace = namespaces.get(2);
+
+    // get a free node port for NGINX
+    nodeportshttp = getNextFreePort(30305, 30405);
+    int nodeportshttps = getNextFreePort(30443, 30543);
+
+    // install and verify NGINX
+    logger.info("Installing and verifying NGINX");
+    nginxHelmParams = installAndVerifyNginx(nginxNamespace, nodeportshttp, nodeportshttps);
+  }
+
+  /**
+   * Create operator and verify it is deployed successfully.
+   * Create domain1 and verify all server pods in domain1 were created and ready.
+   * Verify NGINX can access the sample app from all managed servers in domain1
+   * Delete operator.
+   * Verify the states of all server pods in domain1 were not changed.
+   * Verify NGINX can access the sample app from all managed servers in domain1.
+   * Test fails if the state of any pod in domain1 was changed or NGINX can not access the sample app from
+   * all managed servers in the absence of operator.
+   */
+  @Test
+  @Order(1)
+  @DisplayName("Create operator and domain, then delete operator and verify the domain is still running")
+  @Slow
+  @MustNotRunInParallel
+  public void testDeleteOperatorButNotDomain() {
+    // install and verify operator
+    logger.info("Installing and verifying operator");
+    HelmParams opHelmParams = installAndVerifyOperator(opNamespace, domainNamespace);
+
+    // create and verify the domain
+    logger.info("Creating and verifying model in image domain");
+    createAndVerifyMiiDomain();
```

Comment:
It is not needed, the extension framework will take care. See the ItMiiDomain.java as a reference.

Reply:
this is a local method. I am using the MII_BASIC_IMAGE_NAME built by our framework. This method is create the MII domain using the pre-built image and verify the pods are ready.